css/index.css

```
@@ -1,3 +1,443 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+  width: 900px;
+  margin: auto;
+}
+
+h1 {
+  font-size: 70px;
+}
+
+h3 {
+  font-size: 24px;
+}
+
+p {
+  line-height: 1.3em;
+}
+
+header {
+  display: flex;
+  align-items: baseline;
+  margin-top: 20px;
+  justify-content: space-between;
+}
+
+header nav a {
+  padding-right: 40px;
+  padding-left: 5px;
+  color: grey;
+  font-size: 16px;
+  text-decoration: none;
+}
+
+header img {
+  padding-left: 40px;
+}
+
+.first-content {
+  width: 100%;
+  margin-top: 80px;
+  margin-bottom: 40px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.top {
+  text-align: center;
+}
+
+button {
+  background-color: white;
+  color: black;
+  border: 1px solid black;
+  padding: 5px 35px;
+  font-size: 16px;
+  margin-top: 15px;
+}
+
+button:hover {
+  background-color: rgb(81, 81, 160);
+  border: 1px solid white;
+  color: white;
+}
+
+.second-content {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+}
+
+.third-content {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 20px;
+}
+
```

```suggestion
width: 100%;
```
Here is the image css I think makes the best responsive image for this situation. It messes up your padding a little, but I think that's an easy fix.